apps/wpcom-block-editor/src/calypso/features/legacy-preview-button.scss

```diff
@@ -12,7 +12,7 @@
 		display: inline-flex;
 	}
 	// Normally visible on desktops, make it always hidden
-	.editor-post-preview__dropdown {
```

Comment:
Should we keep the old class too for compatibility with older Gutenberg versions? This might be the case for Atomic when the plugin is disabled for example.

Reply:
Oh dang, you're right. Thanks for the comment!